src/components/FeedbackWidget.tsx

```diff
@@ -154,18 +154,20 @@ const FeedbackWidget: React.FC<IProps> = ({ className }) => {
   const intl = useIntl()
   const containerRef = useRef<HTMLInputElement>(null)
   useOnClickOutside(containerRef, () => handleClose(), [`mousedown`])
+  const [location, setLocation] = useState("")
   const [isOpen, setIsOpen] = useState<boolean>(false)
   const [feedbackSubmitted, setFeedbackSubmitted] = useState<boolean>(false)
   const [isHelpful, setIsHelpful] = useState<boolean | null>(null)
 
-  const location = typeof window !== "undefined" ? window.location.href : ""
```

Comment:
This line was causing hydration problems. On server side this was generating a different output than in client side.